### DISCUSSION TITLE
Ability to use common text editing shortcuts

### DISCUSSION THREAD

**Discussion by Lanhild** [growilabs/growi#8807] [Ideas / Requests]
See https://wsgrowi.slack.com/archives/C01U5NBHR5F/p1715642292449559

**Comment by yuki-takei** [comment]
Loan Joliveau wrote:

> We should have the ability to use common text editing shortcuts in the editor, such as CTRL+B for bold text, CTRL+I for italic, etc.

**Comment by yuki-takei** [comment]
## References
- https://discuss.codemirror.net/t/overriding-certain-default-keymaps/6181
- https://zenn.dev/team_zenn/articles/zenn-markdown-editor-by-cm6

## Current customization
https://github.com/weseek/growi/blob/v7.0.4/packages/editor/src/components/CodeMirrorEditorMain.tsx#L49

**Reply by Lanhild** [reply]
I see that `CTRL`+`S` is already existing. What about `CTRL`+`B`, etc., for example? Would it be possible to add these shortcuts?

**Comment by yuki-takei** [comment]
This request overlaps with the feature idea [**More Keyboard Shortcuts #11143**](https://github.com/growilabs/growi/discussions/11143) proposed by the GROWI development team.

Please upvote and share your thoughts there!
